Jenkinsfile

```diff
@@ -4,15 +4,17 @@ pipeline {
         timeout(time: 1, unit: 'HOURS')
     }
     agent {
-        docker {
-            image 'maven:3.5.0-jdk-8'
+        dockerfile {
```

Comment:
As the main author of `docker-workflow`, I do not trust it at all, which is why in #15 I am ceasing to use it.